CHANGELOG.md

```diff
@@ -1,3 +1,47 @@
+1.1.0-RC / 2021-02-03
+==================
+
+This is a release candidate of 1.1.0 version. Note that final 1.1.0 version may include more features and bugfixes, 
+which would be listed in the corresponding changelog.
+
+### Kotlin version requirement updated
+
+Due to changes in calling conventions between compiler plugin and serialization core runtime, this release requires 
+Kotlin version at least 1.4.30-M1. However, this changes should not affect your code,
+because only deprecated functions were removed from public API.
+See [corresponding PR](https://github.com/Kotlin/kotlinx.serialization/pull/1260) for the details.
+
+### Experimental support for inline classes (IR only)
+
+Using 1.1.0-RC, you can mark inline classes as `@Serializable` and use them in other serializable classes. 
+Unsigned integer types (`UByte`, `UShort`, `UInt` and `ULong`) are serializable as well and have special support in JSON.
+This feature requires Kotlin compiler 1.4.30-RC and enabling new IR compilers for for [JS](https://kotlinlang.org/docs/reference/js-ir-compiler.html) and [JVM](https://kotlinlang.org/docs/reference/whatsnew14.html#new-jvm-ir-backend).
+
+You can learn more in the [documentation](docs/inline-classes.md)
+and corresponding [pull request](https://github.com/Kotlin/kotlinx.serialization/pull/1244).
+
+### Other features
+
+  * Add serializerOrNull function for KType and Type arguments (#1164)
```

Comment:
Here and below I suggest wrapping all methods/classes into backticks, so they are properly rendered